[2025-01-17 00:56:24,830] {taskinstance.py:897} INFO - Dependencies not met for <TaskInstance: agile_data_science_batch_prediction_model_training.pyspark_train_classifier_model 2016-12-26T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2025-01-16T23:56:24.830408+00:00 and task will be retried at 2025-01-16T23:59:27.702375+00:00.
[2025-01-17 00:56:24,831] {local_task_job.py:96} INFO - Task is not able to be run
[2025-01-17 00:56:58,655] {taskinstance.py:897} INFO - Dependencies not met for <TaskInstance: agile_data_science_batch_prediction_model_training.pyspark_train_classifier_model 2016-12-26T00:00:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2025-01-16T23:56:58.655256+00:00 and task will be retried at 2025-01-16T23:59:27.702375+00:00.
[2025-01-17 00:56:58,657] {local_task_job.py:96} INFO - Task is not able to be run
